pkgs/development/compilers/llvm/common/default.nix

```diff
@@ -483,7 +483,18 @@ let
               stripLen = 1;
               hash = "sha256-fqw5gTSEOGs3kAguR4tINFG7Xja1RAje+q67HJt2nGg=";
             })
-          ];
+          ]
+          ++
+            lib.optional (lib.versions.major metadata.release_version == "20")
+              # Fix for vt_gen, remove during next git bump
+              # https://github.com/llvm/llvm-project/pull/109817
+              (
+                fetchpatch {
+                  url = "https://github.com/llvm/llvm-project/commit/af97bd22fd75eb84f27b0106ae7a7080dd06669b.patch";
```

Comment:
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

This probably means this isn't a stable URL — I know this is only going to be around for a week, but it's still nice for people to be able to build old Nixpkgs in the future.

Reply:
It's the commit hash of that PR which was merged upstream, idk how it wouldn't be a part of upstream after being merged.

Reply:
It looks like it was squash merged.

Reply:
So we should use https://github.com/llvm/llvm-project/commit/cace9869775a185793122f845d81a5ff46f15728 instead?

Reply:
Yeah.

Reply:
Or just update to latest git.

Reply:
I updated the patch, didn't feel like bumping to a latest git and trying to get this done fully before work. Also nice to have the branch and commit match lol.

Reply:
That's fine.